com.glitterware.passy.yml

```diff
@@ -0,0 +1,39 @@
+app-id: com.glitterware.passy
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: passy
+separate-locales: false
+finish-args:
+- '--share=ipc'
+- '--socket=fallback-x11'
+- '--socket=wayland'
+- '--device=dri'
+- '--socket=pulseaudio'
+- '--share=network'
+- '--filesystem=home'
```

Comment:
why does a password manager require full *HOME* access?

Reply:
No longer requires full home access: 6c9ee5f7810d38754097f8f545756c88e57342ea. It needs access to the Documents directory, because it stores its data in the `Passy` subdirectory in there.